packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -347,11 +347,17 @@ export class EditorStore implements CommandRegistrar {
     if (graphEditor === GRAPH_EDITOR_MODE.GRAMMAR_TEXT) {
       // Stop change detection as we don't need the actual change detection in text mode
       this.changeDetectionState.stop();
-      this.changeDetectionState.computeLocalChangesInTextMode(
-        (yield this.graphManagerState.graphManager.pureCodeToEntities(
-          this.grammarTextEditorState.graphGrammarText,
-        )) as Entity[],
-      );
+      try {
+        yield flowResult(
+          this.changeDetectionState.computeLocalChangesInTextMode(
+            (yield this.graphManagerState.graphManager.pureCodeToEntities(
+              this.grammarTextEditorState.graphGrammarText,
+            )) as Entity[],
+          ),
+        );
+      } catch (error) {
+        assertErrorThrown(error);
```

Comment:
Could we console this error as a warning. 
Also could you explain how this fixes the problem. Does it allow something else to happen after this method. 
My guess is we are now able to set the current revision entities (in the video case the dataspace and model), so that way when user recompiles it is compared it against those entities.

Reply:
yeah now it is able to set the entityIndexes. When user recompiles it compares against that entityIndexes.